siliconcompiler/tools/vpr/xml_constraints_gen.py

```diff
@@ -0,0 +1,107 @@
+import xml.etree.ElementTree as ET
```

Comment:
maybe rename this file as _xml_constraint.py to help indicate it's not a task but a utility file